docs/Contributing.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
```

Comment:
rename to `CONTRIBUTING.md` (caps) and link to this file from the README

Reply:
I wanted to keep all the docs in the same format with camel case titles, for consistency. I know its not how its typically done, but I like it a bit better this way.

Added the link

Reply:
I dont think the link is present anymore